docs/docs/behaviors/mouse-emulation.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Mouse Emulation Behaviors
+sidebar_label: Mouse Emulation
+---
+
+## Summary
+
+Mouse emulation behaviors send mouse events. Currently, only mouse button presses are supported, but movement
+and scroll action support is planned for the future.
+
+## Configuration Option
+
+This feature can be enabled via a config option:
+
+```
+CONFIG_ZMK_MOUSE=y
+```
+
+If you use the mouse key press behavior in your keymap, the feature will automatically be enabled for you.
+
```

Comment:
```suggestion
```
Do you think we should remove this section altogether? It doesn't sound relevant to the end users if it gets automatically enabled.